src/common/components/ImageUpload/ImageUpload.vue

```diff
@@ -51,40 +62,45 @@ async function onUpdateImage (files: FileList) {
     onUpdate(files[0])
   }
 }
+const modelValue = defineModel<File | null>()
 </script>
 
 <template>
   <AvFileUpload
+    v-model:error="imageUpload.error.value"
+    v-model:valid-message="imageUpload.valid.value"
+    :file-upload-title="t('global.information.fileUpload.title')"
+    :file-upload-description="t('global.information.fileUpload.dragAndDrop')"
```

Comment:
Comme on est déjà dans des propriétés de `AvFileUpload`, tu peux directement les appeler `title` et `description`

```suggestion
    :title="t('global.information.fileUpload.title')"
    :description="t('global.information.fileUpload.dragAndDrop')"
```